doc/intro.rst

```diff
@@ -98,6 +98,26 @@ Example with test data as regular ``POST`` parameters::
         --form attachment=@/path/to/extra-info.txt \
         https://squad.example.com/api/submit/my-group/my-project/x.y.z/my-ci-env
 
+Example with test data using Python's requests library:
+
+.. code:: python
+    import json
+    import requests
+
+    tests = json.dumps({"test1": "pass", "test2": "fail"})
+    metrics = json.dumps({"metric1": 21, "metric2": 4})
+    metadata = json.dumps({"foo": "bar", "baz": "qux"})
```

Comment:
I think metadata needs some keys we consider 'mandatory'

Reply:
@mwasilew I reviewed the code that handles metadata, and no fields are actually mandatory. The only auto-filling we do is regarding job_id: https://github.com/Linaro/squad/blob/master/squad/core/tasks/__init__.py#L146

What we do have is `SPECIAL_FIELDS`: https://github.com/Linaro/squad/blob/master/squad/core/tasks/__init__.py#L117, but I don't think it's necessary to add them all to the examples. What do you think?

Reply:
OK, let's not use special fields.